source/globalCommands.py

```diff
@@ -3304,13 +3309,42 @@ def script_braille_toggleTether(self, gesture):
 		# (braille can be tethered automatically or to either focus or review position).
 		ui.message(_("Braille tethered %s") % TetherTo(newTetherChoice).displayString)
 
+	@script(
+		# Translators: Input help mode message for toggle braille mode command
+		description=_('Cycles through the available braille modes'),
+		category=SCRCAT_BRAILLE,
+		gesture="kb:nvda+alt+t"
+	)
+	def script_toggleBrailleMode(self, gesture):
+		curMode = BrailleMode(config.conf['braille']['mode'])
+		modeList = list(BrailleMode)
+		index = modeList.index(curMode)
+		index = index + 1 if not index == len(modeList) - 1 else 0
+		newMode = modeList[index]
+		config.conf['braille']['mode'] = newMode.value
+		if braille.handler.buffer == braille.handler.messageBuffer:
+			braille.handler._dismissMessage()
+		braille.handler.mainBuffer.clear()
+		# Translators: The message reported when switching braille modes
+		message = f'{_("Braille mode")} {newMode.displayString}'
+		ui.message(message)

```

Comment:
This will not be translated correctly

```suggestion
		# Translators: The message reported when switching braille modes
		ui.message(_("Braille mode {brailleMode}")).format(brailleMode=newMode.displayString)
```